docs/en/enterprise-edition/policy-reference/azure-policies/azure-general-policies/bc-azure-244.adoc

```diff
@@ -0,0 +1,49 @@
+
+== Local users used for Azure Storage
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/azure/StorageLocalUsers.py[CKV_AZURE_244]
+
+|Severity
+|LOW
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy is checking for the use of local users within Azure Storage accounts. Local users can pose a security risk because they might have direct access to the storage account, which can potentially lead to unauthorized data access or compromise if those credentials are exposed or not managed securely. The best practice is to use Azure Active Directory (AD) identities or managed identities to handle access control. This is preferred because Azure AD provides advanced security features, centralized management, and stronger authentication methods, reducing the risk of unauthorized access compared to local users that rely solely on static credentials.
```

Comment:
This policy detects whether local users are used within Azure Storage accounts. Local users can pose a security risk as they may have direct access to the storage account, which could lead to unauthorized data access or compromise if their credentials are exposed or not managed securely. The best practice is to use Azure Active Directory (AD) identities or managed identities for access control. Azure AD offers advanced security features, centralized management, and stronger authentication methods, reducing the risk of unauthorized access compared to local users, which rely solely on static credentials.